aten/src/ATen/Declarations.cwrap

```diff
@@ -69,35 +69,41 @@
 [[
   name: _th_masked_fill_
   cname: maskedFill
+  cpu_bool: True
```

Comment:
can you separate this out in another PR (or just throw it away?).  This has nothing to do with masking, you are just enabling these functions for bool values, but they don't support half values (what's the difference?) and there is no larger audit or what should be supported (presumably index_ functions should also be supported?  i.e. everything that doesn't support masking).